lib/types/local-date.js

```diff
@@ -172,20 +210,12 @@ class LocalDate {
      * if the given one is greater.
      */
     compare(other) {
-        const thisValue = isNaN(this.date.getTime())
-            ? this._value * millisecondsPerDay
-            : this.date.getTime();
-        const otherValue = isNaN(other.date.getTime())
-            ? other._value * millisecondsPerDay
-            : other.date.getTime();
-        const diff = thisValue - otherValue;
-        if (diff < 0) {
-            return -1;
-        }
-        if (diff > 0) {
+        if (this.valueSinceEpoch == other.valueSinceEpoch) {
+            return 0;
+        } else if (this.valueSinceEpoch > other.valueSinceEpoch) {
             return 1;
         }
-        return 0;
+        return -1;
```

Comment:
Why did you droped using `isNan`. Didn't it change output?

Reply:
This used `isNaN` because `_value` returned `NaN` in some cases. The new member returns value always. This does not change output of function.